react.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+	extends: ["airbnb", "airbnb/hooks"],
+	plugins: [],
+	env: {
+		es6: true,
+		node: true,
+		browser: true,
+	},
+	parserOptions: {
+		ecmaVersion: 6,
+		ecmaFeatures: {
+			jsx: true,
+		},
+		sourceType: "script",
+	},
+	rules: {
+		indent: [2, "tab"],
+		"prettier/prettier": "error",
+		"react/jsx-uses-vars": "error",
+		"react/prop-types": ["error", { ignore: ["children"] }],
+		"comma-dangle": [2, "always-multiline"],
+		"object-shorthand": [
+			2,
+			"always",
+			{
+				ignoreConstructors: false,
+				avoidQuotes: false, // this is the override vs airbnb
```

Comment:
You know what... Maybe instead of writing this let's have a comment for every override explaining what it's doing. What do you think?

Reply:
yeah, I added a explanation, for this rule